packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -77,7 +78,7 @@ describe('Dropdown', () => {
       )
     })
 
-    it('calls onOpen with highlightedIndex - 1 on arrow down', () => {
+    it('has the provided prop value - 1 when opened by ArrowUp', () => {
```

Comment:
nit: lets try to avoid the terminology that is code specific, like `ArrowUp` - we might better use implementation-agnostic `arrow up key`